ubuntu/packages

```diff
@@ -52,3 +52,9 @@ r-cran-rcpp
 r-cran-rinside
 srm-ifce-dev
 unixodbc-dev
+libxxhash-dev
+libfcgi-dev
+libboost-dev
+wget
+doxygen
+texlive-font-utils
```

Comment:
+1 for libxxhash-dev and libfcgi-dev, since they are actually dependencies of ROOT.

However, I'd prefer to keep the image as light as possible, so I'd not add doxygen or texlive-font-utils. What's the reasoning for installing those?

Reply:
I'd say they are also dependencies of ROOT, seeing that they are needed to render the ROOT documentation, no?

Reply:
Yes, but it depends on what we will use the image for. If most people are using the image to run ROOT, but look up the documentation online, then it's better to let them install these by hand when using the image for generating the documentation, to keep the image size down, as it's already quite large.

Reply:
The main reason why I think this would be a useful addition is that for people who have ROOT-based packages or frameworks, the image becomes a lot more useful when adding doxygen, as in their CI, they can then use the same image to compile their package and to render their documentation.
If you think that doxygen blows up the image too much, we can of course remove it, but I think it would be useful to provide an additional image with doxygen in that case for CI use.

Reply:
I think you need the equivalent of `tex-gyre-fonts` on Ubuntu. This is required for correct creation and display of histograms, etc. ROOT doesn't embed correctly otherwise. If that is provided by `tex-font-utils` then ok.

You also want a new line at the end of that file.

Reply:
Yes, as far as I can see, this is exactly what is provided by `tex-font-utils`. If you know of a more specific, smaller package that provides them, let me know, I'll exchange it in that case.